js/src/modals/ExportAccount/ExportAccount.js

```diff
@@ -0,0 +1,182 @@
+// Copyright 2015-2017 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import { observer } from 'mobx-react';
+import React, { Component, PropTypes } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import { newError } from '~/redux/actions';
+import { personalAccountsInfo } from '~/redux/providers/personalActions';
+import { AccountCard, Button, Portal, SelectionList } from '~/ui';
+import { Input } from '~/ui/Form';
+import { CancelIcon, CheckIcon } from '~/ui/Icons';
+import ExportStore from './ExportStore';
```

Comment:
I would import, and create an instance per Component here. (Aligns with what we do elsewhere as well), i.e.

```
... extends Component {
  ...
  exportStore = new ExportStore();
  ...
  render () {
    const { canExport, ... } = this.exportStore;
...
}
```